Compression/src/CompressDigiMCs_module.cc

```diff
@@ -119,16 +151,24 @@ class mu2e::CompressDigiMCs : public art::EDProducer {
 
 private:
 
-  // art tags for the input collections
+  Config _conf;
+
   art::InputTag _strawDigiMCTag;
   art::InputTag _crvDigiMCTag;
-
   std::vector<art::InputTag> _simParticleTags;
   std::vector<art::InputTag> _extraStepPointMCTags;
   std::vector<art::InputTag> _timeMapTags;
+  art::InputTag _caloClusterMCTag;
+  art::InputTag _crvCoincClusterMCTag;
+  art::InputTag _primaryParticleTag;
+  art::InputTag _mcTrajectoryTag;
+  bool _keepAllGenParticles;
+  art::InputTag _strawDigiMCIndexMapTag;
+  art::InputTag _crvDigiMCIndexMapTag;
   std::vector<art::InputTag> _caloShowerStepTags;
   art::InputTag _caloShowerSimTag;
-  art::InputTag _CaloShowerROTag;
```

Comment:
same comment as before about CaloShowerSim.  Also CaloShowerRO isn't kept so doesn't need compression in our new workflows.